llvm/lib/Transforms/Scalar/IR2Vec-LOF/custom_loop_distribution/Readme.md

```diff
@@ -0,0 +1,109 @@
+
+# Loop Distribution
+
+This directory contains the Python model related files for training and inferencing Loop Distribution.
+
+
+
+This repo contains the source code and relevant information described in the [paper](https://ieeexplore.ieee.org/abstract/document/10026979)
+Please see [here](https://compilers.cse.iith.ac.in/publications/rl_loop_distribution/) for more details.
+
+> Reinforcement Learning assisted Loop Distribution for Locality and Vectorization, Shalini Jain, S. VenkataKeerthy, Rohit Aggarwal, Tharun Kumar Dangeti, Dibyendu Das, Ramakrishna Upadrasta
+
+
+## Environment Setup
+
+
+Setup the environment using the `ml-llvm-project/model/LoopDistribution/src/LOF_original_env.yml` file with the following command
+
+```bash
+conda env create -f <path-to-the-yml-file>
+```
+
+
+## Inference on Trained Models
+
+Currently three kinds of model runners are supported by us, via the `MLCompilerBridge` 
+- [ONNX Model Runner](#inference-flow-using-onnx-model-runner) - In-Process Model Runner
+- [gRPC Model Runner](#inference-flow-using-grpc-model-runner) - gRPC Based Model Runner
+- [Pipe Model Runner](#inference-flow-using-pipe-model-runner) - Pipe based Model Runner
+
+`Note`: Run ONNX based Model Runner for inference flow to get light and fast compilation.
+
+## Inference flow using gRPC Model Runner
```

Comment:
Is this flow working? Remove it if it is not.

Reply:
Yes GRPC is working fine.